docs/release-notes/whats-new-3.rst

```diff
@@ -299,6 +299,22 @@ OpenAPI schema as YAML has been moved from the default dependencies to the
 ``litestar[yaml]`` package extra.
 
 
+Removal of ``litestar.contrib.htmx`` module
+-------------------------------------------
+
+The deprecated ``litestar.contrib.htmx`` module has been removed. All HTMX functionality
+is now available in :mod:`litestar.plugins.htmx`, which provides the same interface
```

Comment:
I don't think that's a good idea. Either it's an internal thing, or not. If it's an external package, we shouldn't re-export it from the `litestar` namespace.